pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerGauge.java

```diff
@@ -53,6 +53,19 @@ public enum ServerGauge implements AbstractMetrics.Gauge {
   NETTY_POOLED_ARENAS_HEAP("arenas", true),
   STREAM_DATA_LOSS("streamDataLoss", false),
 
+  // Segment operation throttle metrics - threshold is the upper limit of the throttle and is set whenever the
+  // throttle configs are modified
+  SEGMENT_TABLE_DOWNLOAD_THROTTLE_THRESHOLD("segmentTableDownloadThrottleThreshold", false),
+  SEGMENT_DOWNLOAD_THROTTLE_THRESHOLD("segmentDownloadThrottleThreshold", true),
+  SEGMENT_ALL_PREPROCESS_THROTTLE_THRESHOLD("segmentAllPreprocessThrottleThreshold", true),
+  SEGMENT_STARTREE_PREPROCESS_THROTTLE_THRESHOLD("segmentStartreePreprocessDownloadThreshold", true),
```

Comment:
remove `Download` from the metrics name?

Reply:
oops, good catch. done